app/scripts/components/home.js

```diff
@@ -78,6 +81,10 @@ class Home extends React.Component {
       [tally(get(this.props.rules, 'list.meta.count')), 'Ingest Rules', '/rules'],
       [seconds(get(stats.data, 'processingTime.value', nullValue)), 'Average processing Time']
     ];
+    const distStats = [
```

Comment:
What happens when there is no data for errors or successes and we just have the initial state? Does that need a component test?

Reply:
The distribution metrics simply don't appear when there is no data.